### ISSUE TITLE
Unsupported mysqldump args cause scripts to fail with MariaDB

### ISSUE THREAD

**Issue by michaelroper** [nystudio107/craft-scripts#59]
Apparently MariaDB does not recognise the `--set-gtid-purged=OFF` arg that was added in the last update, so `pull_db.sh` may fail if the DB in use is MariaDB. 
```sh
$ ./scripts/pull_db.sh
mysqldump: unknown variable 'set-gtid-purged=OFF'
mysqldump: unknown variable 'set-gtid-purged=OFF'
site-db-dump-20210206.sql.gz       100%   58     0.9KB/s   00:00    
mysqldump: unknown variable 'set-gtid-purged=OFF'
```
Found a [similar issue on the Laravel repo](https://github.com/laravel/framework/issues/32552) which probably describes it better.

If I comment out [that line](https://github.com/nystudio107/craft-scripts/blob/0794d7e81c533bc549bf672787fa70f1ac4cbf0f/scripts/common/common_mysql.sh#L34), it seems to work fine for me again, though my remote is a MariaDB instance on Amazon RDS. Not sure if there's a way around it that also works for those permissions issues you were trying to solve?

**Comment by khalwat** [comment]
hmmm. Probably the way to solve it would be to move an additional params to the `.env.sh` file or such.

**Comment by rungta** [comment]
We are running into this issue with MariaDB on our production servers as well. Realized that DB backups were just empty files. Going to the previous commit d8ccc74217ccd2df0be05bfe779efd5c5a5db648 fixes the problem.

**Comment by khalwat** [comment]
Fixed in https://github.com/nystudio107/craft-scripts/releases/tag/1.2.13

**Comment by khalwat** [comment]
Fixed in 1.2.13 https://github.com/nystudio107/craft-scripts/releases/tag/1.2.13
